README.md

```diff
@@ -921,7 +921,7 @@ Other Style Guides
     // bad
     [1, 2, 3].map(number => {
       const nextNumber = number + 1;
-      `A string containing the ${nextNumber}.`;
+      return `A string containing the ${nextNumber}.`;
```

Comment:
This is intentional; a) the example is trying to show the hazard of omitting `return` in block-body form, b) it's fine if bad examples violate the guide in more than one way.